src/main/resources/feedbackQuestionRankResultStatsRecipientTemplateSelfResponse.html

```diff
@@ -0,0 +1,32 @@
+<div class="panel-body">
+    <div class="row">
+        <div class="col-sm-4 text-color-gray">
+            <strong>
+                Response Summary
+            </strong>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-12">
+            <table class="table table-bordered table-responsive margin-0">
+                <thead>
+                <tr>
+                    <td class="button-sort-none toggle-sort" onclick="toggleSort(this);">Team
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-ascending toggle-sort" id="button_sortteamname" style="width: 35%;">${optionRecipientDisplayName}
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none toggle-sort" id="button_sortname" style="width:15%;">Ranks Received
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none toggle-sort" id="button_sortclaimed" style="width:15%;">Average Rank
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none toggle-sort" id="button_sortrank" style="width:15%;">Average Rank Excluding Self
+                        <span class="icon-sort unsorted"></span></td>
+                </tr>
```

Comment:
I was not talking about the `toggleSort` (but it's good that you spotted it!); I was talking about the formatting.